config/routes.rb

```diff
@@ -12,6 +12,12 @@
   get '/api/openid_connect/userinfo' => 'openid_connect/user_info#show'
   post '/api/risc/security_events' => 'risc/security_events#create'
 
+  unless IdentityConfig.store.idv_api_enabled
```

Comment:
This should be `if`? 

```suggestion
  if IdentityConfig.store.idv_api_enabled
```